dockstore-webservice/src/main/java/io/dockstore/webservice/core/Notification.java

```diff
@@ -0,0 +1,94 @@
+package io.dockstore.webservice.core;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import io.swagger.annotations.ApiModel;
+
+/**
+ * Represents a site-wide notification that will be displayed in a notification bar at the top of every page
+ *
+ * @since 1.8.0
+ */
+
+
+@ApiModel("Notification")
+@Entity
+@Table(name = "notification")
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.Notification.getActiveNotifications",
+                query = "SELECT n FROM Notification n WHERE n.expiration > CURRENT_TIMESTAMP"),
+})
+public class Notification {
+
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.IDENTITY)  // id is auto incremented by the database
+    private long id;
+
```

Comment:
Missing the automatically managed `@CreationTimestamp` and `@UpdateTimestamp` which can be useful for debugging 
https://github.com/dockstore/dockstore/blob/0a8e996843e0c7385a928c0cdc2f506c8a9382e2/dockstore-webservice/src/main/java/io/dockstore/webservice/core/Event.java#L78